src/components/datagrid/data_grid.tsx

```diff
@@ -66,11 +117,34 @@ export class EuiDataGrid extends Component<EuiDataGridProps, EuiDataGridState> {
       rowCount,
       renderCellValue,
       className,
+      gridStyle = {},
       ...rest
     } = this.props;
 
+    const fontSize = gridStyle.fontSize ? gridStyle.fontSize : 'm';
```

Comment:
Can this not be applied as default props in the instantiation? 

If not, you can also simplify some of these ternary functions like:

```suggestion
    const fontSize = gridStyle.fontSize || 'm';
```